controllers/home.js

```diff
@@ -30,9 +34,31 @@ async function home(req, res) {
     return res.redirect("/user/dashboard");
   }
 
+  // Rewrites the /share/{COLOR} links to /
+  if (req.session.redirectHome) {
+    req.session.redirectHome = false;
+    return res.redirect("/");
+  }
```

Comment:
Before this flag, if a user went to `/share/{non-breach-identifier}`, it would leave that as the URL. This puts them back to the homepage, proper like. 

Note that if you share a breach detail link (Ex: `share/Zynga`), it keeps the `/share/` pattern.